phoebe/frontend/default_bundles/default_binary.bundle

```diff
@@ -426,8 +426,8 @@ null
 "description": "Source for bolometric limb darkening coefficients (used only for irradiation; 'auto' to interpolate from the applicable table according to the 'atm' parameter, or the name of a specific atmosphere table)",
 "choices": [
 "auto",
-"phoenix",
-"ck2004"
+"ck2004",
+"phoenix"
```

Comment:
Should these edits be a part of this particular bugfix?

Reply:
@MichaelAbdul-Masih - these can safely be excluded from the diff if you want, but also don't do any harm (I think the order when exporting to json is somewhat random and so this often shows up when regenerating the default cached bundles).